Detectors/FIT/simulation/src/Digitizer.cxx

```diff
@@ -0,0 +1,100 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "FITSimulation/Digitizer.h"
+
+#include "TCanvas.h"
+#include "TFile.h"
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TLeaf.h"
+#include "TMath.h"
+#include "TProfile2D.h"
+#include "TRandom.h"
+#include <algorithm>
+#include <cassert>
+
+using namespace o2::fit;
+
+ClassImp(Digitizer);
+
+void Digitizer::process(const std::vector<HitType>* hits, std::vector<Digit>* digits)
+{
+
+  // hits array of FIT hits for a given simulated event
+  mDigits = digits;
+  Double_t timeframe = 0;
+  Int_t bc = 0;
+  Int_t ampthreshold = 100;
+  Float_t lowTimeA = 10000, lowTimeC = 2500, highTimeA = 12500, highTimeC = 4500;
+  Int_t mcp, trackID;
+  Int_t amp[208];
+  Double_t cfd[208];
+  for (Int_t ipmt = 0; ipmt < 208; ipmt++) {
+    amp[ipmt] = 0;
+    cfd[ipmt] = 0;
+  }
+  for (auto& hit : *hits) {
+    // TODO: put timeframe counting/selection
+    // if (timeframe == mTimeFrameCurrent) {
+    // timeframe = Int_t((mEventTime + hit.GetTime())); // to be replaced with uncalibrated time
+    mcp = hit.GetDetectorID();
+    if ((mcp < 96 && hit.GetTime() > lowTimeA && hit.GetTime() < highTimeA) ||
+        (mcp > 95 && hit.GetTime() > lowTimeC && hit.GetTime() < highTimeC)) {
```

Comment:
what do these constants `95` etc mean?

Reply:
Hello
MCPs from 0-95 are from A side, 95-207 - C side. Pisition of T0A is 320cm, T0C  -80cm